test/utils.ts

```diff
@@ -178,3 +178,7 @@ export async function getViewerStatus(pdfFilePath: string) {
         }
     }, process.platform === 'win32' ? 600 : undefined)
 }
+
+export function checkDictkeys(keys: string[], expectedKeys: string[], optKeys: string[] = []): boolean {
+    return keys.every(k => expectedKeys.includes(k) || optKeys.includes(k)) && expectedKeys.every(k => keys.includes(k))
+}
```

Comment:
- `check` is too general. `assertEqualDictKeys` is better.
- Move to `unittest.test.ts`. The function is only for the test.
- We had better call `assert.deepStrictEqual`: https://nodejs.org/api/assert.html#assertdeepstrictequalactual-expected-message

Reply:
>  The word `check` is too general. It is not clear what is asserted.

I have turned this function into an assertion directly `assertDictKeyNames`. I hope this is clearer.